frontend/src/Student/components/ShowCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import FaPlusCircle from 'babel-loader!react-icons/fa/plus-circle'
+import { Button, ButtonGroup, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+
+const ShowCard = (props) => (
+  <Card>
+    <Row>
+      <Col>
+        <h2>{props.name}</h2>
```

Comment:
I would align this text so the baseline is the same as "Submission Ends"

Reply:
You want it centered with  Accepting Submissions Until correct?